tests/testcases/030_check-network.yml

```diff
@@ -12,19 +12,25 @@
       bin_dir: "/usr/local/bin"
     when: not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
 
+  - name: Check kubectl output
+    shell: "{{bin_dir}}/kubectl get pods --all-namespaces -owide"
+    register: get_pods
+
+  - debug: msg="{{get_pods.stdout}}"
```

Comment:
I can certainly take it out, but it really helps you get an idea of what is running and its status. The ping failure isn't so useful if you can't see what is running.